sprokit/processes/vxl/kw_archive_writer_process.cxx

```diff
@@ -147,7 +166,7 @@ kw_archive_writer_process
 }
 
 
-// ----------------------------------------------------------------
+//-----------------------------------------------------------------------------------
```

Comment:
Note: KWIVER prefers to keep the space here...

Reply:
Updated